src/requirements.md

```diff
@@ -14,12 +14,10 @@ Note: The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “
   - The client library SHOULD be architected as to eliminate coupling altogether.
 
 - The client MUST support multiple platforms to improve the security of wallets on mobile devices/web/desktop.
-  - The client MUST be a command-line application with a support of linux target
-    - TODO: Mac? Windows?
+  - The client MUST be a command-line application with a support of linux target. The client MUST be implemented in a cross-platform way and SHOULD compile against Mac OS and Windows platforms however full Windows and Mac OS supoprt is out of the scope and can be added within further development.
   - The client library MUST be usable on popular mobile platforms (iOS, Android)
   - The client library MUST be usable in browsers as a WebAssembly module, with a JS wrapper (hereafter _WASM Module_).
-  - The client library MUST support linux target
-    - TODO: Mac? Windows?
+  - The client library MUST support linux target. It MUST be implemented in a cross-platform way, but full support of other platforms (e.g. Windows, Mac OS, Android) is out of the scope.
```

Comment:
again I'd stay positive: "It SHOULD be implemented in a cross-platform way", period